cli/cli.py

```diff
@@ -0,0 +1,282 @@
+import argparse
+import csv
+import sys
+import json
+import io
+import tabulate
+import uuid
+
+# This script will implement cli for Studio public API.
```

Comment:
Let's remove this comment, and extend README.md with a
```
## Command-line interface (CLI)
```
section that talks about what it is and how to use it in a few sentences.

Reply:
👍